enterprise_access/apps/subsidy_access_policy/models.py

```diff
@@ -421,6 +425,18 @@ def get_content_price(self, content_key, content_metadata=None):
             raise ContentPriceNullException(f'The price for {content_key} is null')
         return content_price
 
+    def get_list_price(self, lms_user_id, content_key):  # pylint: disable=unused-argument
+        """
+        Determine the price for content for display purposes only.
+        We likely have content metadata prefetched on this policy record instance at the time
+        of invocation, so we do that prefetch here via ``self.get_content_metadata()``.
+        """
+        return get_list_price_for_content(
+            self.enterprise_customer_uuid,
+            content_key,
+            self.get_content_metadata(content_key),
```

Comment:
The memoization makes this cheap, as long as `get_list_price()` is called _after_ `can_redeem()` was called on this policy.

Reply:
strike that - it's really request caching here.  There's no longer memoization going on (outside of `RequestCache`).